api/app/resources/bookings/exam/exam_export_list.py

```diff
@@ -82,68 +82,63 @@ def get(self):
+                    elif key == "booking_id":
+                        row.append(exam.booking.booking_id)
+                    elif key == "booking_name":
+                        row.append(exam.booking.booking_name)
+                    elif key == "exam_received_date":
                         if exam.exam_received_date is None:
-                            if exam.exam_returned_ind == 0:
-                                out.writerow([exam.office.office_name, exam.exam_type.exam_type_name, exam.exam_id, exam.exam_name,
-                                              exam.examinee_name,
-                                              exam.event_id, exam.booking.room.room_name, exam.booking.invigilator.invigilator_name,
-                                              exam.booking.booking_id, exam.booking.booking_name, 'N',
-                                              'N'])
-                            else:
-                                out.writerow([exam.office.office_name, exam.exam_type.exam_type_name, exam.exam_id,
-                                              exam.exam_name,
-                                              exam.examinee_name,
-                                              exam.event_id, exam.booking.room.room_name, exam.booking.invigilator.invigilator_name,
-                                              exam.booking.booking_id, exam.booking.booking_name, 'N',
-                                              'Y'])
+                            row.append("N")
                         else:
-                            if exam.exam_returned_ind == 0:
-                                out.writerow([exam.office.office_name, exam.exam_type.exam_type_name, exam.exam_id, exam.exam_name,
-                                              exam.examinee_name,
-                                              exam.event_id, exam.booking.room.room_name, exam.booking.invigilator.invigilator_name,
-                                              exam.booking.booking_id, exam.booking.booking_name, 'Y',
-                                              'N'])
-                            else:
-                                out.writerow([exam.office.office_name, exam.exam_type.exam_type_name, exam.exam_id,
-                                              exam.exam_name,
-                                              exam.examinee_name,
-                                              exam.event_id, exam.booking.room.room_name, exam.booking.invigilator.invigilator_name,
-                                              exam.booking.booking_id, exam.booking.booking_name, 'Y',
-                                              'Y'])
-
+                            row.append("Y")
+                    elif key == "exam_returned_ind":
+                        if exam.exam_returned_ind == 0:
+                            row.append("N")
+                        else:
+                            row.append("Y")
+                try:
```

Comment:
Is this try / catch still valid? I'm wondering how `out.writerow` would throw an AttributeError, and if it did, why the response is for "Invigilator Name Not Found".

Reply:
fixed, tested and pushed up